marxs/source/labSource.py

```diff
@@ -92,5 +109,7 @@ def generate_photons(self, t):
                         np.sin(theta) * np.cos(phi),
                         np.sin(phi),
                         np.zeros(n)])
-
-        return Table({'pos': pos.T, 'dir': dir.T, 'energy': np.ones(n).T, 'polarization': np.random.uniform(0, 2 * np.pi, n).T, 'probability': np.ones(n).T})
+                        
+        polarization = self.random_polarization(n, dir.T)
```

Comment:
OK, here is a complicated idea. I don't think we need that at this point (if there is not need for added complexity, then why make it more complicated then necessary?), but I want to write that down as a "cool things you can do in python" kind of tip: Consider the following 
```
def random_polarization(n, dir_array):
    ... stuff here as above ...

class Source(...):
    polarization_func = random_polarization
    ... other stuff here...
   def generate_photons(...):
       ...
       pol = self.polarization(m, dir.T)
```
If I want a more complicated polarization I can do:
```
s = Source()
s.polarization = my_complicated_polarization_function
```
as long as my function accepts the same arguments.